configure.ac

```diff
@@ -281,6 +282,8 @@
             CFLAGS="${CFLAGS} -fPIC"
             RUST_LDADD="-ldl -lrt -lm"
             can_build_shared_library="yes"
+            AC_DEFINE([SYSTEMD_NOTIFY], [1], [make Suricata notify systemd on startup])
+            enable_systemd="yes"
```

Comment:
There is a community of no-systemd distros out there.

Reply:
Guess we can leave it mostly as is, as the env var check and the socket checks could just quietly fail. Should remove the warning for those cases. Not really interested in adding a compile time option for it. In distros packaging it can easily be patched out

Reply:
I guess no headers are included, so probably safe to leave as-is.